src/Agent/NewRelic/Agent/Core/WireModels/LogEventWireModel.cs

```diff
@@ -70,5 +86,30 @@ public LogEventWireModel(long unixTimestampMS, string message, string level, str
             TraceId = traceId;
             Priority = priority;
         }
+
+        public LogEventWireModel(long unixTimestampMS, string message, string level, ICollection<string> errorStack, string errorMessage, string errorClass, string spanId, string traceId)
+        {
+            TimeStamp = unixTimestampMS;
+            Message = message.TruncateUnicodeStringByBytes(MaxMessageLengthInBytes);
+            Level = level;
+            SpanId = spanId;
+            TraceId = traceId;
+            ErrorStack = string.Join(" \n", errorStack);
```

Comment:
Are there other places where we concatenate multi-line text? If so, maybe it would make sense to have a helper function to do the formatting?

Reply:
Other stack traces are sent up as an array, not a single string - the results from the ScrubAndTruncate call.  Unfortunately, we also need to keep the stack lines to no more than 300 so we use the existing ScrubAndTruncate  to get the lines truncated and in the same format as out existing stacks and then join it back to a single string when we build the model.

Given its a such a small bit of code, I don;t think a helper function makes much sense here.